test/test.js

```diff
@@ -8,7 +8,11 @@ import { strictEqual, deepStrictEqual } from 'assert'
 import reader from './utilities/reader.js'
 const require = createRequire(import.meta.url)
 const readFile = util.promisify(fs.readFile)
-// const writeFile = util.promisify(fs.writeFile)
+const writeFile = util.promisify(fs.writeFile)
+
+const sortBy = (fn) => {
+  return (a, b) => (fn(a) > fn(b)) ? 1 : ((fn(b) > fn(a)) ? -1 : 0);
+}
```

Comment:
I tend to pull in https://underscorejs.org/ for this and other niceties but this works just fine :-)